website/companies_view.html

```diff
@@ -0,0 +1,89 @@
+<!doctype html>
```

Comment:
Not sure what exactly this page was intended to look like. I think it's trying to show the overview of all companies. Some graphs that Kenny is working on could be useful to display in this page?